core-bundle/src/Resources/contao/classes/Versions.php

```diff
@@ -217,6 +217,8 @@ public function create($blnHideUser=false)
 			$strDescription = $data['subject'];
 		}
 
+		$strDescription = StringUtil::substr($strDescription, System::getContainer()->get('database_connection')->getSchemaManager()->listTableColumns('tl_version')['description']->getLength());
```

Comment:
Judging from the doc block `StringUtil::substr()` strips tags and may return something slightly longer than the given `$intNumberOfChars`. Is this the intended outcome here?

Reply:
> The function preserves words, so the result might be a bit shorter or longer than the number of characters given. It strips all tags.

I think, the length must not be longer than the detected length of the DB field.

@m-vo Any suggestion to get the ticket finally done?

Reply:
> and may return something slightly longer than the given `$intNumberOfChars`.

If this is the case we should fix `StringUtil::substr()` IMO to never return something longer than `$intNumberOfChars`.

Reply:
Thank you @ausi . That was my thought too.

Reply:
If you don‘t need the special functionality of the `StringUtil::substr()` method, please just use `mb_substr()` directly.

Reply:
Seems the version title is currently not displayed anywhere. So, the special functionality of the `StringUtil::substr()` method is not needed. I changed to code to `mb_substr()`.